code/src/vs/workbench/browser/parts/titlebar/windowTitle.ts

```diff
@@ -32,7 +32,8 @@ import { CodeWindow } from '../../../../base/browser/window.js';
 
 const enum WindowSettingNames {
 	titleSeparator = 'window.titleSeparator',
-	title = 'window.title'
+	title = 'window.title',
+	headerText = 'window.headerText'
```

Comment:
Could we truncate property name to `window.header` ?

Reply:
OK for me!
will provide the corresponding changes...